packages/viewer/src/mathml.ts

```diff
@@ -48,6 +48,41 @@ function decodeSafeMathML(root: HTMLElement) {
   }
 }
 
+export async function renderAMathML(properties: Properties, mathElement: MathMLElement): Promise<void> {
```

Comment:
I would like to delete the `renderMathML` function, but it seems to have a lot of implications and requires many document modifications. However, it appears worth discussing its feasibility.

Reply:
I would like so too. I would even consider this part of the development of the task, since the `renderMathML` is used for the retro-compatibility methods, which could benefit from the new performance improvements.

It would require a part of investigation and maybe adapt the current methods so that `renderMathML` can be properly replaced.